.storybook/assets/index.css

```diff
@@ -66,6 +67,15 @@ nav .spectrum-Site-logo {
 	padding: 20px !important;
 }
 
+/* Hide that first divider line in the top navigation */
+div.sb-bar > div > div > span:first-child {
```

Comment:
Is there a more stable way to select this? It feels a bit fragile. ✨

Reply:
Oh it's insanely fragile but it's basically all we have access to in Storybook without overwriting the base components they're using. All the classes are randomly generated.